### ISSUE TITLE
Scrollable cannot be disabled in vertical mode

### ISSUE THREAD

**Issue by lookouttt** [prabhuignoto/react-chrono#338]
**Describe the bug**
The scrollbar appears on my vertical timeline by default.  When I try to remove it by declaring it false nothing happens.

**To Reproduce**
Here is a copy of how I am creating the Chrono component
<Chrono items={albumItems} mode="VERTICAL" cardHeight={"10px"} allowDynamicUpdate='true' useReadMore theme={{
                                            primary: '#ce7f2f',
                                            secondary: '#4A4A4A',
                                            cardBgColor: '#a57038',
                                            cardForeColor: 'white',
                                            titleColor: 'white',
                                            titleColorActive: '#ed8b2a',
                                            }}/>

I tried to use scrollable = 'false'.  I also tried scrollable = {{scrollbar: false}}.  Neither option removed the scrollbar.

**Expected behavior**
Based on the documentation, I don't expect the scrollbar to be visible unless I set the scrollable value to true.  I am seeing the scrollbar using the code listed above.

**Screenshots**
![ChronoScrollbar](https://user-images.githubusercontent.com/116034747/221755797-236e519e-1fc8-4de4-ac02-2bcdb010c4e5.png)

**Desktop (please complete the following information):**
Chrome Version 110.0.5481.177

**Additional context**
I actually don't mind the scrollbar, but I'm finding that my timeline items are not dynamically resizing like the examples I've seen on your site or code sandbox.  So I was trying to remove the scrollbar to see if that was killing the resizing capability.  When I tried that, I found it was not working.  So I have not been able to troubleshoot the resizing issue.

**Comment by lookouttt** [comment]
After looking into this further, it appears what happens is if the timeline is placed inside a div that is smaller than the timeline it will automatically add the scroll bars.  In this instance it also does not resize correctly as you make the screen smaller.  When I take it out of the div, the timeline resizes correctly.  If this is as you intend, then I suppose it's not a bug.  But I will need to find a way for it to both be in the div and resize correctly.

**Comment by prabhuignoto** [comment]
@lookouttt thanks for looking further in to this issue. would you be able to share the code via codesandbox or by any other means

**Comment by lookouttt** [comment]
@prabhuignoto  - Thank you for the quick reply.  I actually just found the issue.  I was using a fixed width on the div and not a maxWidth.  Once I changed that most things seem to be working correctly.  The only thing I'm not seeing is the "Read More" popping up as the boxes get smaller.  I am using the useReadMore prop.  But I will take more time to do my own research first.

I apologize that I jumped the gun on posting this.  I'm an aspiring web developer, and I'm trying to finish up a project I'm working on.  I will close this item.
